styles/style.css

```diff
@@ -0,0 +1,95 @@
+.container {
+  background-color: #e5e5e5;
+  width: 389px;
+}
+
+nav {
+  width: 20.188rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 1.5rem;
+}
+
+.flex {
+  display: flex;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
```

Comment:
[Optional]
My recommendation is, to use CSS classes to style the HTML elements instead of directly styling the HTML elements. Because CSS classes are reusable and easy to maintain. 😃